frontend/src/common/components/header/components/SearchFilter.tsx

```diff
@@ -136,6 +137,9 @@ export const SearchFilter = () => {
       .map(([key, value]) => `${key}=${value}`)
       .join("&");
 
+    if (data?.searchEntity === SEARCH_ENTITIES.PERMIT && data?.searchValue?.trim()?.length < 1)
+      return;
+
```

Comment:
maybe leave out the `searchEntity` condition and just return for ANY search that has no given query value?